### PR TITLE
State api

### PR DESCRIPTION
**Pull request by KRLoyd**
Pull request to merge state_api to master

### CODE REVIEW

File: api/v1/views/index.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 app = Flask(__name__)
 
 
-@app_views.route("/status")
+@app_views.route("/status", strict_slashes=False)
```
Comment (kjowong) [inline api/v1/views/index.py]: Are you going to add for the rest of the routes?

File: api/v1/views/states.py [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+        abort(404)
+    # Get dict of attributes in request
+    json_data = request.get_json()
+    # Check json_data
+    if not json_data:
+        return jsonify({"message": "Not a JSON"}), 400
+    # Create list of keys we don't want updated
+    ignore_keys = ['id', 'created_at', 'updated_at']
+    # Loop through the json_data, update/add keys
+    for attribute, value in json_data.items():
```
Comment (kjowong) [inline api/v1/views/states.py]: For attribute - is it better to use key for consistency with rest of codebase? I think this is fine, but just a thought

**Review by kjowong — CHANGES_REQUESTED** [review]
Good code! Maybe some better variable names?

**Comment by KRLoyd** [comment]
Changes have been made, per your requests. 
You are totally right about "attribute" needing to be renamed to "key"

**Review by kjowong — APPROVED** [review]

**Pull request merged**